### DISCUSSION TITLE
🪐 Universe 2023 - Copilot transforming GitHub

### DISCUSSION THREAD

**Discussion by Akash1134** [community/community#74693] [Announcements]
Today [we're announcing](https://github.blog/2023-11-08-universe-2023-copilot-transforms-github-into-the-ai-powered-developer-platform/) the general availability of GitHub Copilot Chat as well as previews of the new GitHub Copilot Enterprise offering, new AI-powered security features, and the GitHub Copilot Partner Program. 

In March, we shared our vision for the future of software development with [Copilot X](https://github.com/orgs/community/discussions/50727), seamlessly integrating AI into the developer lifecycle. Since then, we've been refining the technology, making our vision a reality. 

Check out this [blog post](https://github.blog/2023-11-08-universe-2023-copilot-transforms-github-into-the-ai-powered-developer-platform/) by GitHub CEO Thomas Dohmke, where he shares how Copilot is transforming GitHub into the AI-powered developer platform. Let's delve into the exciting developments:

![Screenshot 2023-11-08 at 10 52 17 PM](https://github.com/community/community/assets/95834130/31bcd954-0ca1-4fd0-a7ed-90a2a20d4bbd)
> Alt Text: The image showcases the key Copilot ships at Universe on a black background

## GitHub Copilot Chat Arriving in December 2023

**Copilot Chat will be available in December 2023 as part of your existing GitHub Copilot subscription, for both individuals and organizations**. Moreover, it's free for verified teachers, students, and maintainers of popular open source projects.

**Copilot Chat is more powerful than ever:**

- **Powered by GPT-4:** We've upgraded Copilot Chat to provide more accurate code suggestions and explanations with OpenAI's GPT-4 model.
- **Code-aware guidance and generation:** Copilot Chat leverages your code as context, offering explanations, code suggestions based on your open files, and even helping detect security vulnerabilities.
- **Inline Copilot Chat:** Developers can now discuss specific lines of code directly within their code editor.
- **Slash commands:** Introducing slash commands and context variables to streamline tasks in GitHub Copilot.
- **Smart actions:** Offering powerful shortcuts to enhance your workflow with a single click.
- **Copilot Chat for JetBrains:** Copilot Chat will soon be available for the JetBrains suite of IDEs.

## GitHub Copilot Chat on github.com and Mobile App

GitHub Copilot Chat is now seamlessly integrated 🔄  into github.com, enhancing the developer experience by providing suggestions, summaries, analysis, and answers. It's also available in our mobile app, allowing developers to get answers and insights on the go.

With Copilot in the code editor, CLI, and now on github.com and the mobile app, we're making Copilot ubiquitous throughout the software development lifecycle.

## GitHub Copilot Enterprise 💼 

GitHub Copilot Enterprise is designed to empower teams of developers to quickly understand your codebase, build documentation, and receive suggestions based on internal and private code. This solution streamlines the development process and enhances security, all with enterprise-grade privacy.

**Pricing for GitHub Copilot Business is $19 per user per month, while GitHub Copilot Enterprise will be available in February 2024 for $39 USD per user per month.**

## GitHub Copilot Partner Program 🤝 

We're supercharging GitHub Copilot by integrating it with third-party developer tools and services, broadening the possibilities for developers. This program will create an ecosystem for new networks and ingenuity to be infused into GitHub Copilot, enabling developers to achieve even more with AI.

[Apply to be an early access GitHub Copilot Partner today](https://gh.io/AAmkjk6)

## New AI-powered Security Features 🔐 
GitHub Copilot now includes AI-powered security features that prevent vulnerable coding patterns in real-time. We've also infused GitHub Advanced Security with new AI-powered application security testing features, enhancing code scanning and secret scanning capabilities.

These features are available today in preview and will soon be included in your GitHub Advanced Security subscription.

[Join the GitHub Advanced Security waitlist today.](https://gh.io/AAmo6qh)

## Coming Next ⏭️: GitHub Copilot Workspace 🏢 

The GitHub Copilot Workspace is designed to help developers bridge the gap between idea and code implementation. It offers automatically proposed plans and allows you to validate and test code changes quickly. Copilot Workspace is a significant step towards a future where developers can leverage AI as a second brain for rapid and creative code generation.

## Bringing It All Together 🤲 

The future of GitHub Copilot is defined by the intersection of human and artificial intelligence 🤖. We are committed to creating a holistic and seamless AI-powered developer platform that empowers developers of all kinds, from startups to enterprises, and open source communities.

[Join us in this exciting journey!
](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=keynote-launch)

[Read more](https://gh.io/AAmo6qh) and stay tuned for the amazing features coming your way. 🚀

**Comment by gitstua** [comment]
To join the excitement you can join discussions with the team:
[Universe 2023 - AI](https://github.com/orgs/community/discussions/categories/ai?discussions_q=is%3Aopen+whatsapp+category%3AAI)
[Universe 2023 - DevEx](https://github.com/orgs/community/discussions/categories/devex?discussions_q=is%3Aopen+whatsapp+category%3ADevEx)
[Universe 2023 - Security](https://github.com/orgs/community/discussions/categories/security?discussions_q=is%3Aopen+whatsapp+category%3ASecurity)

**Reply by AleksRG** [reply]
Thanks @gitstua !

**Reply by CoderNitu** [reply]
Informative 👍

**Comment by mahesh-s33** [comment]
GitHub Copilot Workspace seems like a Gamechanger! Cant wait for it's release. :)

**Reply by gitstua** [reply]
Thanks @mahesh-s33 - looks like you are as excited as we are as we use AI to improve Developer Experience 🚀

**Reply by mahesh-s33** [reply]
Thanks!

**Reply by mishmanners** [reply]
Yasss I can't wait 🙌

**Reply by jurban1997** [reply]
Agreed.  I think I'm the target customer.  I'm a beginner at software development, but I've got decades of process design and product design experience.  I've already pushed Chat GPT to its limit for building a site.  I need an IDE that spans all files and helps me work through requirements while minimizing incompatible variables and maintaining consistency in the site architecture.  If I can do this, I'll avoid having to raise $M to get the idea off the ground.

**Comment by mishmanners** [comment]
Really excited to see this! 🥳 🚀

**Reply by AleksRG** [reply]
Me too!

**Comment by Kingodbug** [comment]
> Today [we're announcing](https://github.blog/2023-11-08-universe-2023-copilot-transforms-github-into-the-ai-powered-developer-platform/) the general availability of GitHub Copilot Chat as well as previews of the new GitHub Copilot Enterprise offering, new AI-powered security features, and the GitHub Copilot Partner Program.
> 
> In March, we shared our vision for the future of software development with [Copilot X](https://github.com/orgs/community/discussions/50727), seamlessly integrating AI into the developer lifecycle. Since then, we've been refining the technology, making our vision a reality.
> 
> Check out this [blog post](https://github.blog/2023-11-08-universe-2023-copilot-transforms-github-into-the-ai-powered-developer-platform/) by GitHub CEO Thomas Dohmke, where he shares how Copilot is transforming GitHub into the AI-powered developer platform. Let's delve into the exciting developments:
> 
> ![Screenshot 2023-11-08 at 10 52 17 PM](https://user-images.githubusercontent.com/95834130/281489646-31bcd954-0ca1-4fd0-a7ed-90a2a20d4bbd.png)
> 
> > Alt Text: The image showcases the key Copilot ships at Universe on a black background
> 
> ## GitHub Copilot Chat Arriving in December 2023
> **Copilot Chat will be available in December 2023 as part of your existing GitHub Copilot subscription, for both individuals and organizations**. Moreover, it's free for verified teachers, students, and maintainers of popular open source projects.
> 
> **Copilot Chat is more powerful than ever:**
> 
> * **Powered by GPT-4:** We've upgraded Copilot Chat to provide more accurate code suggestions and explanations with OpenAI's GPT-4 model.
> * **Code-aware guidance and generation:** Copilot Chat leverages your code as context, offering explanations, code suggestions based on your open files, and even helping detect security vulnerabilities.
> * **Inline Copilot Chat:** Developers can now discuss specific lines of code directly within their code editor.
> * **Slash commands:** Introducing slash commands and context variables to streamline tasks in GitHub Copilot.
> * **Smart actions:** Offering powerful shortcuts to enhance your workflow with a single click.
> * **Copilot Chat for JetBrains:** Copilot Chat will soon be available for the JetBrains suite of IDEs.
> 
> ## GitHub Copilot Chat on github.com and Mobile App
> GitHub Copilot Chat is now seamlessly integrated 🔄 into github.com, enhancing the developer experience by providing suggestions, summaries, analysis, and answers. It's also available in our mobile app, allowing developers to get answers and insights on the go.
> 
> With Copilot in the code editor, CLI, and now on github.com and the mobile app, we're making Copilot ubiquitous throughout the software development lifecycle.
> 
> #### [Join the Copilot Chat waitlist today](https://gh.io/AAmo6qh)
> ## GitHub Copilot Enterprise 💼
> GitHub Copilot Enterprise is designed to empower teams of developers to quickly understand your codebase, build documentation, and receive suggestions based on internal and private code. This solution streamlines the development process and enhances security, all with enterprise-grade privacy.
> 
> **Pricing for GitHub Copilot Business is $19 per user per month, while GitHub Copilot Enterprise will be available in February 2024 for $39 USD per user per month.**
> 
> #### [Join the Copilot Enterprise waitlist today](https://gh.io/AAmo6qh)
> ## GitHub Copilot Partner Program 🤝
> We're supercharging GitHub Copilot by integrating it with third-party developer tools and services, broadening the possibilities for developers. This program will create an ecosystem for new networks and ingenuity to be infused into GitHub Copilot, enabling developers to achieve even more with AI.
> 
> [Apply to be an early access GitHub Copilot Partner today](https://gh.io/AAmkjk6)
> 
> ## New AI-powered Security Features 🔐
> GitHub Copilot now includes AI-powered security features that prevent vulnerable coding patterns in real-time. We've also infused GitHub Advanced Security with new AI-powered application security testing features, enhancing code scanning and secret scanning capabilities.
> 
> These features are available today in preview and will soon be included in your GitHub Advanced Security subscription.
> 
> [Join the GitHub Advanced Security waitlist today.](https://gh.io/AAmo6qh)
> 
> ## Coming Next ⏭️: GitHub Copilot Workspace 🏢
> The GitHub Copilot Workspace is designed to help developers bridge the gap between idea and code implementation. It offers automatically proposed plans and allows you to validate and test code changes quickly. Copilot Workspace is a significant step towards a future where developers can leverage AI as a second brain for rapid and creative code generation.
> 
> ## Bringing It All Together 🤲
> The future of GitHub Copilot is defined by the intersection of human and artificial intelligence 🤖. We are committed to creating a holistic and seamless AI-powered developer platform that empowers developers of all kinds, from startups to enterprises, and open source communities.
> 
> [Join us in this exciting journey! ](https://githubuniverse.com/?utm_source=Discussion&utm_medium=GitHub&utm_campaign=keynote-launch)
> 
> [Read more](https://gh.io/AAmo6qh) and stay tuned for the amazing features coming your way. 🚀

mabuhay' master of the pipeline*

**Comment by Kingodbug** [comment]
mabuhay

**Comment by nicomulderlawvu1** [comment]
Does Copilot Chat will be available in December 2023 mean 1 Dec or some time Dec 2023? We would like to enable it, but waiting for GA due to the T's and C's for beta releases.

**Comment by AlejandrosTesla8** [comment]
😆👍🏽

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
